sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 1.0.0-beta.3 (Unreleased)
-
+* [ApplicationInsightsSampler introduced for compatibility with Application Insights SDKs. Implements the same hash algorithm](#22288).
```

Comment:
Probably a nit but just want to call it out loud here - 

This line would render as "Sentence A. Sentence B.", with "Sentence A. Sentence B" showing as a link and the end period (`.`) NOT part of the link - which might be strange.

What I guess we might consider:
1. Only put the link at the very end, rather than making the entire line as a link ```ApplicationInsightsSampler introduced for compatibility with Application Insights SDKs. Implements the same hash algorithm. (#22288)```
2. Include the ending period in the link ```[ApplicationInsightsSampler introduced for compatibility with Application Insights SDKs. Implements the same hash algorithm.](#22288)```

Reply:
Great suggestions, I totally see that. Thinking of incorporating both suggestions in a way:

ApplicationInsightsSampler introduced for compatibility with Application Insights SDKs, as it implements the same hash algorithm to decide to sample. (#22288)

Reply:
That looks great to me 👍